pkg/application/componentInstallationUseCase.go

```diff
@@ -20,6 +20,10 @@ const (
 		"`allow-switch-distribution-namespace` flag, issue a feature request"
 )
 
+var (
+	noActionErr = fmt.Errorf("fail because the total amount of actions is zero. actions should contain at least action %q if nothing changed", domain.ActionNone)
```

Comment:
🚫 **[checkstyle]** <errname> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>the variable name `noActionErr` should conform to the `errXxx` format